_posts/2018-10-21-nabla-k8s.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: post
+title: Nabla on Kubernetes!
+---
+
+A while back, we released a demo of the nabla containers runtime, `runnc` (v0.1). Most recently, we have improved our runtime in several aspects such as better start-up times and memory density. In addition to that (and also the focus of this blogpost), we have implemented Nabla support for kubernetes! In this blogpost, we will show a simple local setup of kubernetes using the nabla containers runtime, `runnc` (v0.2), through [containerd](https://github.com/containerd/containerd)  via the [CRI untrusted workload cri plugin and pod annotation](https://github.com/containerd/cri/blob/master/pkg/annotations/annotations.go).
+
+**Announcement: We will presenting Nabla Containers at Kubecon China 2018, and a joint presentation with Katacontainers at Kubecon Seattle 2018, so do come by and chat with us!**
+
+There will be several steps to achieve this on a local machine:
+
+1. Build and install Nabla runtime `runnc`
+2. Setup containerd with `runnc`
+3. Setup a local kubernetes cluster with containerd
+4. Setup a CNI config
+5. Running Nabla on k8s!
+
+We note for this post, the following versions were used:
+```
+ubuntu: 16.04.3 LTS
+- kernel: 4.15.0-24-generic
+runnc: v0.2
+- commit: 1df1a09c891decaf0bc4279c2322126932863c96
+containerd: v1.2.0-rc1
+- commit: 0c5f8f63c3368856c320ae8a1c125e703b73b51d
+kubernetes: v1.11.0
+- commit: 91e7b4fd31fcd3d5f436da26c980becec37ceefe
+```
+
+##  Build and install Nabla runtime `runnc`
+
+```
+Runnc Version Info: v0.2 (commit: 1df1a09c891decaf0bc4279c2322126932863c96)
+```
+
+The first step to running nabla on kubernetes is to run nabla! The first order of business is to make sure we have a running new (v0.2) version of `runnc`. For a step-by-step guide to install `runnc`, take a look at our previous article: [Running a Nabla Container]({% post_url 2018-06-28-nabla-setup %}).
```

Comment:
also add a pointer "or look at the README".  I think over time the README should be the authoritative source that stays up to date